### DISCUSSION TITLE
蓝灯专业版手机不能连接

### DISCUSSION THREAD

**Discussion by Lewis0000221** [getlantern/lantern-forum-cn#842] [General]
专业版刚买了一个星期不够，体验感越来越差，电脑倒是能上，但手机的体验一天比一天差，甘开通前两天网速还可以，到后面那几天一时可以连一时不能连上，今天还干脆用不了了，卸载重新安装后免费版反而能连上，体验超级差，节点不停切换，居然没有固定节点可选

**Comment by Lewis0000221** [comment]
麻烦请退款，
账号：andylewis@163.com
![微信图片_20240907140828](https://github.com/user-attachments/assets/e2e73046-f703-428d-9cd9-b19c15323256)

**Comment by MaxMKMax** [comment]
@Lewis0000221 您好. 很抱歉影响了您的体验感. 好的, 我帮您提交申請退款. 既然是刚买的, 那可以退款.

**Reply by Lewis0000221** [reply]
你好，请问退款了吗？

**Reply by Lewis0000221** [reply]
@Mike777Khoo 请问现在的退款都不理啦？就这样的服务？？？手机不能用也不修复

**Comment by greySkiesMarsh** [comment]
@Lewis0000221 您好，很遗憾听到您在使用 Lantern Pro 时遇到的问题。您的退款正在处理中，应该会在 5-10 个工作日内反映在您的帐户中。

**Reply by Lewis0000221** [reply]
手机APP的专业版根本连不上，退出账号用免费版反而能上，两台手机都是这个情况，那买专业版有什么用呢？

**Reply by Lewis0000221** [reply]
@greySkiesMarsh 请问现在的退款都不理啦？就这样的服务？？？

**Reply by Lewis0000221** [reply]
@greySkiesMarsh 请问退款进度现在什么情况？一个月过去了依然没有退回来

**Reply by Lewis0000221** [reply]
@greySkiesMarsh @Mike777Khoo 请跟进退款进度！！！！现在都不管啦？？？？没有一个回复！！！能骗一个是一个吗？？？
